source/scpp/src/util/Logging.h

```diff
@@ -13,7 +13,17 @@
 #define INFO "info"
```

Comment:
Since we are already modifying this module, it would make sense to also replace these defines with:

```
# define TRACE 0
# define DEBUG 0
# define INFO 1
```

And then you don't have to do expensive string comparisons inside of `DLogger::DLogger`, and can get rid of the `LogTrace / LogInfo / etc` defines.